### PR TITLE
Updated German lang

### PR DESCRIPTION
**Pull request by Stargamers**
Update German lang

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -214,6 +214,7 @@
     <string name="turn_off">Ausschalten</string>
     <string name="unauthorized_user">Unauthorisierter Benutzer</string>
     <string name="two_factors_otp_error">Das einmalige Zwei-Faktor Passwort wird benötigt</string>
+	<string name="no_issue">Kein Issue</string>
```
Comment (yakov116) [inline app/src/main/res/values-de/strings.xml]: Missing an `s` should be `no_issues`

Comment (Stargamers) [reply app/src/main/res/values-de/strings.xml]: Fixed

Comment (yakov116) [reply app/src/main/res/values-de/strings.xml]: Great. Will merge soon.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -214,6 +214,7 @@
     <string name="turn_off">Ausschalten</string>
     <string name="unauthorized_user">Unauthorisierter Benutzer</string>
     <string name="two_factors_otp_error">Das einmalige Zwei-Faktor Passwort wird benötigt</string>
+	<string name="no_issues">Kein Issue</string>
```
Comment (yakov116) [inline app/src/main/res/values-de/strings.xml]: Travis Error
`Found item String/no_issues more than one time`

**Pull request merged**